lib/index.coffee

```diff
@@ -14,9 +15,27 @@ dockerMtimeStream()
 		current_mtimes = layer_mtimes
 
 garbageCollect = (reclaimSpace) ->
-	dokcerImageTree()
-	.then(annotateTree.bine(null, current_mtimes))
+	dockerImageTree()
+	.then(annotateTree.bind(null, current_mtimes))
 	.then(lruSort)
 	.then (candidates) ->
 		# Remove candidates until we reach `reclaimSpace` bytes
-		console.log('foobar')
+		# Candidates is a list of images, with the least recently used
+		# first in the list
+
+		# Decide on the images to remove
+		# TODO: Find a lodash function for this!
+		size = 0
+		remove = []
+		for i in candidates
```

Comment:
These are admittedly only small loops, but would it be better to use `image` for the loop variable names instead of `i`?
(personally, I only tend to use `i` as the loop variable when it's referring to integers)